modules/packages/CopyAggregation.chpl

```diff
@@ -112,10 +112,22 @@ module CopyAggregation {
     type elemType;
     @chpldoc.nodoc
     var agg: if aggregate then DstAggregatorImpl(elemType) else nothing;
+    /* Sets ``dst = srcVal`` in a way that aggregates such updates
+       to improve communication efficiency assuming that ``dst`` is remote
+       and ``srcVal`` is local. */
     inline proc ref copy(ref dst: elemType, const in srcVal: elemType) {
```

Comment:
Again, also not your problem, but it seems weird that this method uses `srcVal` as the name and the other one uses just `src`.  I suppose that's because one uses `in` intent and the other uses `ref`?  I'm not sure how I feel about argument names varying based on the intent use (and that's probably a conversation to have rather than a decision to make between the two of us anyways).

Anyways, that probably should be considered "Lydia musing aloud" rather than an action item for you.

Reply:
I think it's intentional to use srcVal here & I don't want to try to change it unless we're reviewing it for stability (which we aren't right now).